docker/test/performance-comparison/entrypoint.sh

```diff
@@ -146,3 +146,7 @@ dmesg -T > dmesg.log
     report analyze benchmark metrics
 
 cp compare.log /output
+
+# Upload images to S3 to use them in html page
+mkdir -p /output/images/svg
+cp./*.svg /output/images/svg/
```

Comment:
```suggestion
cp ./*.svg /output/images/svg/
```

(as suggested in https://clickhouse-test-reports.s3.yandex.net/24549/fcd679f7cfd5736e7ff524df4b464086c755cc1d/style_check.html#fail1)